docs/scenarios.md

```diff
@@ -0,0 +1,55 @@
+### Scenarios
+Kraken currently supports pod and node based scenarios. 
+
+
```

Comment:
Should we add something here linking to the powerfulseal documentation on how to set up pod scenarios? Maybe link: https://powerfulseal.github.io/powerfulseal/pods-rescheduled

Reply:
Done.